util/create-brackets-table.raku

```diff
@@ -1,74 +1,102 @@
 #!/usr/bin/env raku
 
-# global array defined in the BEGIN block at the end:
-my @bracket-chars;
+# TODO
+#   + get desired exit behavior from Coke
+#   + settle on table name and output file name
+
+# Global array initially defined in the BEGIN block at the end.
+# The array is then defined dynamically by reading the source
+# file at https://github.com/Raku/nqp/src/HLL/Grammar.nqp.
+our @bracket-chars;
 
-my $f = "../doc/Language/brackets.pod6";
 if !@*ARGS {
     say qq:to/HERE/;
-    Usage: {$*PROGRAM.IO.basename} go | reorder
+    Usage: {$*PROGRAM.IO.basename} go | Reorder [refresh][original][debug]
```

Comment:
Let's pick an order and stick with it. Supporting 3 different order types when we'll only use one is adding maintenance for this script for no value - this would remove the need for this option entirely. (If we keep the option: please use lowercase "reorder" making it easier to type.